Sources/AndroidContext.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  AndroidContext.swift
+//  UIKit
+//
+//  Created by Chetan Agarwal on 07/06/2019.
+//  Copyright © 2019 flowkey. All rights reserved.
+//
+
+#if os(Android)
+
+import Foundation
+import JNI
+
+/**
+ https://developer.android.com/reference/android/content/Context
+ */
+class AndroidContext: JNIObject {
```

Comment:
note: we do already have `JavaContext` as a type in the JNI (JavaParameterConvertible+Objects.swift:64) but there are good reasons not to use it here. Specifically, that type is set up as a struct (which doesn't seem right to me) and there doesn't seem to be much further benefit in using it at the moment (the protocol conformances are not very useful to us here unfortunately).

In the medium term, it may make sense to move `AndroidContext` and `AndroidFile` out of UIKit though. Rather than putting them into the JNI, I think a separate module / repo à la `import Android` would be the best for these kinds of higher-level APIs – that would give others a solid base to use and expand on for some fundamental Android functionality that would go beyond (and maybe replace) what `Foundation` gives us.